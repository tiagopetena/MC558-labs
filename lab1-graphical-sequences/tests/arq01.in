10
8 8 6 5 5 5 5 4 4 2