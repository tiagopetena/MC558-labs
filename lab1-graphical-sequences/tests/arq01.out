2 3 4 5 6 7 8 9
1 3 4 5 6 7 8 9
1 2 4 5 6 7
1 2 3 5 6
1 2 3 4 6
1 2 3 4 5
1 2 3 8 9
1 2 7 10
1 2 7 10
8 9
